.github/workflows/tests.yml

```
@@ -18,13 +18,25 @@ jobs:
                     - '8.2'
                     - '8.3'
                     - '8.4'
-                    - '8.5'
                 dependencies: [highest]
+                symfony-require: ['']
                 include:
                     - php-version: '8.0'
                       dependencies: lowest
+                    - php-version: '8.1'
+                      symfony-require: '5.4.*'
+                    - php-version: '8.2'
+                      symfony-require: '6.4.*'
+                    - php-version: '8.3'
+                      symfony-require: '7.2.*'
+                    - php-version: '8.4'
+                      symfony-require: '8.0.*'
+                      stability: dev
 
-        name: PHP ${{ matrix.php-version }} ${{ matrix.dependencies }}
+        name: PHP ${{ matrix.php-version }} ${{ matrix.symfony-require && format('/ Symfony {0}', matrix.symfony-require) || '' }} ${{ matrix.dependencies == 'lowest' && '/ lowest' || '' }}
```

I think the lowest build will have 2 spaces instead of one.

Just do
```suggestion
name: PHP ${{ matrix.php-version }} ${{ matrix.dependencies }} ${{ matrix.symfony-require && format('/ Symfony {0}', matrix.symfony-require) || '' }}
```